tests/Tests/100__deploy/100__installation/103__grafana.robot

```diff
@@ -0,0 +1,86 @@
+*** Settings ***
+Documentation       Post install test cases that verify OCP Grafana resources and objects
+
+Library             OpenShiftLibrary
+Resource            ../../../Resources/ODS.robot
+
+*** Test Cases ***
+Verify Grafana Is Shipped And Enabled Within ODS
+    [Documentation]    Verify Grafana Is Shipped And Enabled Within ODS
+    [Tags]    Sanity
+    ...       Smoke
```

Comment:
I would recommend to add it to Sanity/Tier1 first and after move it to Smoke once is tested for a release

Reply:
done